pulp_rpm/app/models.py

```diff
@@ -325,14 +325,14 @@ def list_to_createrepo_c(l):
             Package model at the moment.
 
             Args:
-                l(list): a list
+                l(list): a string representation of a list
 
             Returns:
                 list: list of strings and/or tuples
 
             """
             createrepo_c_list = []
-            for item in l:
+            for item in json.loads(l):
```

Comment:
it was changed due to: https://pulp.plan.io/issues/5671 and https://pulp.plan.io/issues/5215
Previously fields were CharField, and its method was needed to convert the strings, but now they are JSONField, so they are not string anymore

Reply:
The provided solution fixed the error mentioned in the issue description. But tests are now not passing.

JSONField does not return string by default. Or am I wrong? In the docs, there is written: "When the value is retrieved from the database, it will be in the format chosen by the custom encoder (most often a string), so you’ll need to take extra steps to convert the value back to the initial data type (Model.from_db() and Field.from_db_value() are two possible hooks for that purpose).".

Reply:
The tests passed when I made the change, actually, they broke before the change as json.loads() requires a string. As you can see here: https://travis-ci.org/pulp/pulp_rpm/jobs/613527770#L1958
`TypeError: the JSON object must be str, bytes or bytearray, not list`